components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/java/org/wso2/carbon/identity/webhook/metadata/api/model/ProfileType.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.webhook.metadata.api.model;
+
+/**
+ * Enum representing the event profile type.
+ */
+public enum ProfileType {
+    
+    /**
+     * WSO2 Events profile.
+     */
+    WSO2_EVENT("wso2-event"),
+
+    /**
+     * CAEP Events profile.
+     */
+    CAEP_EVENT("caep-event"),
```

Comment:
Rather than defining the profile type here, I think we should look for a way where we can get supported event profiles from the config file it self.